docs/04-operation-guides/troubleshooting/smsh-03-istio-no-sidecar.md

```diff
@@ -24,5 +29,4 @@ If a Pod doesn't have a sidecar and you did not disable sidecar injection on pur
 
    Sidecar injection is disabled if the output contains the `sidecar.istio.io/inject:false` line. Delete the label and restart the Pod to enable sidecar injection for the Deployment.
 
-
-For more information, read [this document](#details-sidecar-proxy-injection) or follow the [Istio documentation](https://istio.io/docs/ops/common-problems/injection/).
+For more information, read [this document](../operations/smsh-01-istio-disable-sidecar-injection.md) or follow the [Istio documentation](https://istio.io/docs/ops/common-problems/injection/).
```

Comment:
Is there any reason to link to this document twice? 🤔 We're linking to it already in the first paragraph. I'd remove one of the links.